libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -1204,7 +1204,7 @@ private void onMediaTapped(@NotNull AztecAttributes attrs, int naturalWidth, int
                 // Display 'cancel upload' dialog
                 AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
                 builder.setTitle(getString(R.string.stop_upload_dialog_title));
-                builder.setPositiveButton(android.R.string.ok, new DialogInterface.OnClickListener() {
+                builder.setPositiveButton(getString(R.string.stop_upload_dialog_button_yes), new DialogInterface.OnClickListener() {
```

Comment:
can we use the `setPositiveButton(int,...`  version of the API? that is, ditch the `getString()` here and just pass the string resource id reference for `R.string. stop_upload_dialog_button_yes`.